cmd/release-controller/http_helper.go

```diff
@@ -702,7 +702,11 @@ func renderInstallInstructions(w io.Writer, mirror *imagev1.ImageStream, tag *im
 		fmt.Fprintf(w, `<p>Download installer and client with:<pre class="ml-4">oc adm release extract --tools %s</pre>`, template.HTMLEscapeString(tagPull))
 		return
 	}
-	fmt.Fprintf(w, `<p><a href="%s">Download the installer</a> for your operating system or run <pre class="ml-4">oc adm release extract --tools %s</pre>`, template.HTMLEscapeString(fmt.Sprintf("https://%s/%s", artifactsHost, tag.Name)), template.HTMLEscapeString(tagPull))
+	pieces := strings.Split(tagPull, ":")
```

Comment:
This isn't safe, you can't assume a pull spec has a ":" (it could be a digest).  You'll need to parse this.

Reply:
@smarterclayton If a digest is passed in, do we want to process it further, or just post the "Download the installer with:..." message?